lib/dogapi/v1/metric.rb

```diff
@@ -59,16 +65,16 @@ def switch_to_single()
 
       def make_metric_payload(metric, points, scope, options)
         begin
-          typ = options[:type] || 'gauge'
+          type = options[:type]
```

Comment:
Let's add the interval param as well 😃 . See https://docs.datadoghq.com/api/?lang=bash#metrics